src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/DirectionsQuestionnaireGenerator.java

```diff
@@ -815,4 +820,21 @@ private WelshLanguageRequirements getWelshLanguageRequirements(DQ dq) {
                 welshLanguageRequirements.getDocuments()).map(Language::getDisplayedValue).orElse(""))
             .build();
     }
+
+    private String getClaimantStatementOfTruth() {
```

Comment:
I think this can be consolidated into one method and change the word to 'claimant' or 'defendant' depending on case. This way we don't need to upkeep two sets of strings and less code duplication.

_I've not tested the below so it might have some problems. There are other ways to do this too, but I'll leave it up to you to decide._

    ```
public static String getStatementOfTruth(String role) {
        return String.format("The %s believes that the facts this claim are true.\n\n\n"
            + "I am duly authorised by the %s to sign this statement.\n\n"
            + "The %s understands that proceedings for contempt of court may be brought against anyone who makes,"
            + "or causes to be made, a false statement in a document verified by a statement of truth without an honest"
            + "belief in its truth.", IntStream.range(0, 3).mapToObj(i -> role).toArray());
    }
```

Reply:
You can also keep the string as a constant on top of the file, it would make the file look tidier.